tutorials/4_provision_and_instrument_a_database/README.md

```diff
@@ -0,0 +1,182 @@
+            "database_root_password": "fake_example_password"
+        }
+    },
+    "globalTags": {}
+}
+```
+
+Here is a summary of each field within secrets:
+
+| Field Name               |  Field Description |
+| ------------------------ | ------------------ |
+| database_password        | used for the service to connect to the database |
+| database_root_password   | specifically for connecting to the database with the root user |
+
+## [Example deployment configuration](database.json)
+
+```json
+{
+    "services": [
+        {
+            "id": "pythontron",
+            "source_repository": "-b main https://github.com/newrelic/demo-pythontron.git",
+            "deploy_script_path": "deploy/linux/roles",
+            "port": 5001,
+            "destinations": [
+                "host1"
+            ],
+            "params": {
+                "delay_start_ms": 10000,
+                "database_user": "[service:mysql:params:database_user]",
+                "database_password": "[credential:secrets:database_password]",
+                "database_host": "[resource:host1:ip]",
+                "database_port": "[service:mysql:port]"
+            }
+        },
+        {
+            "id": "mysql",
+            "source_repository": "-b main https://github.com/newrelic/demo-services.git",
+            "deploy_script_path": "deploy/mariadb/linux/roles",
+            "port": 6002,
+            "destinations": [
+                "host1"
+            ],
+            "params": {
+                "database_user": "demotron",
+                "database_password": "[credential:secrets:database_password]",
+                "database_root_password": "[credential:secrets:database_root_password]"
+            }
+        }
+    ],
+    "resources": [
+        {
+            "id": "host1",
+            "provider": "aws",
+            "type": "ec2",
+            "size": "t3.micro"
+        }
+    ],
+    "instrumentations": {
+        "resources": [
+            {
+                "id": "nr_infra",
+                "resource_ids": [
+                    "host1"
+                ],
+                "provider": "newrelic",
+                "source_repository": "-b main https://github.com/newrelic/demo-newrelic-instrumentation.git",
+                "deploy_script_path": "deploy/linux/roles",
+                "version": "1.14.2"
+            }
+        ],
+        "services": [
+            {
+                "id": "mysql_integration",
+                "service_ids": [
+                    "mysql"
+                ],
+                "provider": "newrelic",
+                "source_repository": "-b main https://github.com/newrelic/demo-newrelic-instrumentation.git",
+                "deploy_script_path": "deploy/databases/mysql/linux/roles",
+                "version": "1.4.0",
+                "params": {
+                    "database_port": "[service:mysql:port]"
+                }
+            }
+        ]
+    }
+}
+```
+
+### Note - Params
+
```

Comment:
Would you add a table here that goes over the different values that the database needs? I see you have the the passwords covered, but I think a similar explanation of each param would help people understand what's going on here.

Reply:
added. lmk if that looks okay.

Reply:
Looks good to me!